Automattic/wp-calypso
# Plans: Allow purchasing of plans on trial from Plan Overview page

This pull request addresses part of https://github.com/Automattic/wp-calypso/issues/1151 by adding a button in the header of the `Plan Overview` page to allow users to purchase the plan they subscribed to as part of a free trial:

![screenshot](https://cloud.githubusercontent.com/assets/594356/11755320/870b9680-a01f-11e5-8776-0a4088d17f34.png)

This pull request also makes sure the page display correctly on small viewports:

![screenshot](https://cloud.githubusercontent.com/assets/594356/11755387/01c41ed8-a020-11e5-8883-14791e7adf07.png)

#### Testing instructions

1. Get a new test account and blog
2. Enable your sandbox and proxy
3. Select a plan by clicking the `Start Free Trial` button on the [`Plans` page](http://calypso.localhost:3000/plans)
4. Click the `Start 14 Day Free Trial` button on the `Secure Payment` page
5. Check that the purchase was successful
6. Navigate back to the `Plans` page and make sure the `Plan Overview` variant is displayed
7. Click the `Purchase Now` button and make sure the corresponding plan is added to the shopping cart

#### Additional notes

I decided to move the `Purchase Now` button above the progress bar for small viewports as [the opposite](https://github.com/Automattic/wp-calypso/issues/1151#issuecomment-163036162) seemed almost impossible to achieve using a flexbox layout.
